### PR TITLE
iOS flick: Support for element flick in the webview

### PR DESCRIPTION
**Pull request by scf2k**
This PR implements flick functionality in the webview when element is specified. Flick starts from the center of the element and finishes at the point calculated using specified offsets. Under the hood the flick() method  is used from the uiauto/mechanic patch.

### CODE REVIEW

**Comment by scf2k** [comment]
@jlipps please take a look

**Comment by scf2k** [comment]
@0x1mason please take a look

**Comment by jlipps** [comment]
Hi @scf2k code looks good! A couple things:

1) I'd really love a test of this somehow in our ios webview specs
2) You've got a commit from @sebv in here, maybe if you rebase and force-push it'll get rid of it.

File: trigger.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 This is used to test github webhooks.
-1
+0
```
Comment (0x1mason) [inline trigger.txt]: What's this?

**Comment by sebv** [comment]
test this please

**Comment by 0x1mason** [comment]
@scf2k Do you think you could write a test for this?

**Comment by scf2k** [comment]
@0x1mason I'm preparing it

**Comment by 0x1mason** [comment]
:+1: great stuff, thanks!

**Comment by scf2k** [comment]
Added a test.

While making it I noticed that translateWebCoords (ex clickWebCoords) works differently in safari and test web view app. There is a condition which checks for iOS8 and in this case `serviceBarHeight` is set to 0, 20 otherwise. This is correct for safari. But in webview app it should be always 0 to calculate proper result value.

**Comment by scf2k** [comment]
@0x1mason please review

**Comment by scf2k** [comment]
@0x1mason @jlipps sorry I have to ping you :D

**Comment by jlipps** [comment]
All the code looks good to me, thanks @scf2k! @0x1mason is running the current release so I'll let him merge it at an appropriate moment.

**Comment by scf2k** [comment]
great thanks! :)

**Pull request merged**